src/Sylius/Behat/Context/Api/Shop/ProductVariantContext.php

```diff
@@ -232,7 +233,7 @@ public function variantAndVariantShouldNotBeDiscounted(ProductVariantInterface .
 
     private function findVariant(?ProductVariantInterface $variant): array
     {
-        $response = $this->client->getLastResponse();
+        $response = $this->client->showByIri(sprintf('/api/v2/shop/product-variants/%s', $variant->getCode()));
```

Comment:
```suggestion
        $this->client->show($variant->getCode());
```
but why do you change that? After your change, the `if` statement below is not needed and I'm not sure if it is a proper change, as this method was probably used by different steps 🤔